data/json/items/generic/casing.json

```diff
@@ -130,13 +130,61 @@
     "volume": "5ml"
   },
   {
-    "id": "40mm_casing",
+    "id": "40x46mm_m212_casing",
```

Comment:
I still think it's kinda weird to be adding this and the M195 casing seeing as they don't have any use. I don't know what the best solution is, though.

Reply:
They'd be usable for different loads that I haven't added here. In another PR I look at survivor crafted non-lethal and utility loads.